### PR TITLE
removes non-functional intra-document links

### PR DESCRIPTION
**Pull request by ablwr**
Resolves formatting issue presented in #145

### CODE REVIEW

**Comment by ablwr** [comment]
To see the differences more clearly, you can check out this branch locally and run `git diff origin --word-diff` to see the difference at the word-level, instead of sentence-level present in this WebUI. It makes it easier to review.

**Comment by robUx4** [comment]
LGTM. I think the RFC might not have internal links at all anyway. If we want to have some back, we might as well do a proper pass on said document (likely after splitting).

**Comment by ablwr** [comment]
Agreed. I can trim out the other few internal links that I demoted to just anchor-tags (but functional ones).

**Comment by dericed** [comment]
An RFC can include links to section headers. I don't think other types of anchors can be used. Each MKV Element is given it's own section header when the whole doc is built so such anchors can be used. But when the RFC is rendered the link appears as a reference according to the section number. For instance in the EBML document

> `Master Elements` MUST NOT use an unknown size unless the `unknownsizeallowed` attribute of their `EBML Schema` is set to true (see [the section on the `unknownsizeallowed` attribute](#unknownsizeallowed)).

is rendered in the RFC as 

> "Master
>   Elements" MUST NOT use an unknown size unless the
 >  "unknownsizeallowed" attribute of their "EBML Schema" is set to true
  > (see Section 14.1.4.10).

File: chapters.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ The type of the menu system is defined by the ChapProcessCodecID parameter. For
 
 ### Matroska Script (0)
 
-This is the case when [ChapProcessCodecID]({{site.baseurl}}/index.html#ChapProcessCodecID) = 0\. This is a script language build for Matroska purposes. The inspiration comes from ActionScript, javascript and other similar scripting languages. The commands are stored as text commands, in UTF-8\. The syntax is C like, with commands spanned on many lines, each terminating with a ";". You can also include comments at the end of lines with "//" or comment many lines using "/* */". The scripts are stored in ChapProcessData. For the moment ChapProcessPrivate is not used.
+This is the case when ChapProcessCodecID = 0\. This is a script language build for Matroska purposes. The inspiration comes from ActionScript, javascript and other similar scripting languages. The commands are stored as text commands, in UTF-8\. The syntax is C like, with commands spanned on many lines, each terminating with a ";". You can also include comments at the end of lines with "//" or comment many lines using "/* \*/". The scripts are stored in ChapProcessData. For the moment ChapProcessPrivate is not used.
```
Comment (dericed) [inline chapters.md]: could consider backtick-quoting elements
> This is the case when ChapProcessCodecID = 0\.

to
> This is the case when the `ChapProcessCodecID Element` = 0\.

File: codec_specs.md [hunk 2]
```diff
@@ -621,39 +621,39 @@ Codec ID: S_TEXT/UTF8
 
 Codec Name: UTF-8 Plain Text
 
-Description: Basic text subtitles. For more information, please look at the [Subtitle specifications]({{site.baseurl}}/subtitles.html).
+Description: Basic text subtitles. For more information, please look at the Subtitle section of the specification.
```
Comment (dericed) [inline codec_specs.md]: Seems pointed enough to be worth a sectional reference

> For more information, see \[the section on the \`Subtitles\`\](#subtitles).

**Comment by ablwr** [comment]
Great, thanks @dericed! I'll make your recommended changes and this can go through review again.

**Comment by ablwr** [comment]
It works! ✨

**Pull request merged**